Formula/root.rb

```diff
@@ -43,12 +43,6 @@ class Root < Formula
 
   skip_clean "bin"
 
-  # Can be removed post 6.22.08
-  patch do
-    url "https://github.com/root-project/root/commit/d113c9fcf7e1d88c573717c676aa4b97f1db2ea2.patch?full_index=1"
-    sha256 "6d0fd5ccd92fbb27a949ea40ed4fd60a5e112a418d124ca99f05f70c9df31cda"
-  end
-
   def install
     # Work around "error: no member named 'signbit' in the global namespace"
     ENV.delete("SDKROOT") if DevelopmentTools.clang_build_version >= 900
```

Comment:
At least for me (on macOS 11), this builds just fine without this line.

Reply:
Yeah I don't think we even allow the `SDKROOT` env to be set in the superenv anymore.

Reply:
chenrui333 Can you please try removing this line? Don't think it will have much (any?) effect, but is likely unneeded. I don't have permissions to edit this for you.

Reply:
Totally forgot @chenrui333 have me access to the forked repo before. :) I'm trying this without this line for starters.

Reply:
Try
```ruby
ENV.delete "CPATH"
```